vms/secp256k1fx/tx.go

```diff
@@ -8,6 +8,10 @@ type Tx interface {
 	UnsignedBytes() []byte
 }
 
+var (
+	_ Tx = &TestTx{}
```

Comment:
What is this doing?

Reply:
This causes this file to fail to compile if `TestTx` doesn't implement the `Tx` interface.